requirements_docs.txt

```diff
@@ -9,7 +9,7 @@ base58==0.2.5
 bitcoin==1.1.42
 blessings==1.6.1
 bpython==0.17
-byteplay3==3.5.0
+bytecode==0.5
```

Comment:
Why do we need this in the docs requirements?

Reply:
I think at some point we would maybe be using the compiler when generating the docs?

Reply:
Not sure about the use-case, but sure :D